packages/forms-web-app/src/lib/is-token-valid.js

```diff
@@ -1,45 +1,54 @@
 const { checkToken } = require('./appeals-api-wrapper');
 const { isTokenExpired } = require('./is-token-expired');
-const { utils, enterCodeConfig } = require('@pins/common');
+const { utils } = require('@pins/common');
 const config = require('../config');
 
 const testConfirmEmailToken = '12345';
 
-const isTokenValid = async (id, token, session) => {
-	let result = {
-		valid: false,
-		action: ''
-	};
-
-	if (!id || typeof id !== 'string' || !token || typeof token !== 'string') {
-		return result;
-	}
-
-	// is test LPA + Test environment + test token
-	// then allow through as a confirm email token
-	if (
+// is test LPA + Test environment + test token
+// then allow through as a confirm email token
+const isTestLPAAndEnvironment = (token, session) => {
```

Comment:
think this should be renamed to include that it is checking the token, or perhaps a suggestion to be separated into 2 functions if name is too unwieldy